packages/office-ui-fabric-react/src/components/Layer/Layer.base.tsx

```diff
@@ -35,27 +33,6 @@ export class LayerBase extends BaseComponent<ILayerProps, {}> {
   private _host: Node;
   private _layerElement: HTMLElement | undefined;
   private _hasMounted: boolean;
-  /**
-   * Used for notifying applicable Layers that a host is available/unavailable and to re-evaluate Layers that
-   * care about the specific host.
-   */
-  public static notifyHostChanged(id: string) {
```

Comment:
thinking about this change, I'll move the methods back and just redirect to the new ones to not break compat